DEPLOY.md

```diff
@@ -88,7 +90,7 @@ Group=nginx
 WorkingDirectory=/home/nginx/oovirtenv/OpenOversight/OpenOversight
 Environment="PATH=/home/nginx/oovirtenv/bin"
 Environment="PGPASS=~/.pgpass"
-ExecStart=/usr/local/bin/gunicorn -w 4 -b 127.0.0.1:4000 app:app 
+ExecStart=/usr/local/bin/gunicorn -w 16 -b 127.0.0.1:4000 --timeout 90 app:app 
```

Comment:
Ok I'm going to close this PR until we get all the changes we want in this release done...